src/Data/Maybe/Base.agda

```diff
@@ -137,3 +131,22 @@ thisM a = maybe′ (these a) (this a)
 
 thatM : Maybe A → B → These A B
 thatM = maybe′ these that
+
+------------------------------------------------------------------------
+-- Deprecated
+
+-- Version 2.1
+-- decToMaybe
+
+open import Relation.Nullary.Reflects using (invert)
+open import Relation.Nullary.Decidable.Core
+  using (_because_; Dec)
+
+decToMaybe : Dec A → Maybe A
+decToMaybe ( true because [a]) = just (invert [a])
+decToMaybe (false because  _ ) = nothing
+
+{-# WARNING_ON_USAGE decToMaybe
+"Warning: decToMaybe was deprecated in v2.1.
+Please use Relation.Nullary.Decidable's decToMaybe instead."
```

Comment:
Why not simply give the full qualified name `Relation.Nullary.Decidable.decToMaybe` instead?